examples/list_of_lists/test/package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "scripts": {
+        "test": "cd .. && ts-node --transpile-only --ignore=false test/test.ts"
+    },
+    "dependencies": {
+        "azle": "github:demergent-labs/azle#kybra_test_reuse",
+        "fast-equals": "4.0.3"
```

Comment:
If you could make this a dev dependency, `npm install fast-equals --save-dev`.

Reply:
I did this in azle, so I don't think we will need it here once azle is merged in